.travis.yml

```diff
@@ -11,3 +11,6 @@ node_js:
 cache:
   directories:
     - node_modules
+after_script:
+  - npm install -g codeclimate-test-reporter
```

Comment:
I wouldn't expect this to be necessary after configuring the `code_climate` addon above. It doesn't look like the package is installed automatically, as I would expect.

https://travis-ci.org/codeclimate/javascript-test-reporter/jobs/135634591#L253

```
$ codeclimate-test-reporter < ./coverage/lcov.info
/home/travis/build.sh: line 45: codeclimate: command not found
```